run_xnat2bids.py

```diff
@@ -0,0 +1,57 @@
+from getpass import getpass
+import glob
+import os
+import shlex
+import shutil
+import subprocess
+from toml import load
+
+# Load config file into dictionary
+arglist = load('x2b_config.toml')
+
+# Extract arglist from dictionary
+for key in arglist:
```

Comment:
Handle ALL possible inputs to xnat2bids. Open question, how to handle extra slurm inputs?